devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy

```diff
@@ -116,19 +125,29 @@ def initialize() {
 }
 
 private addChildButtons(numberOfButtons) {
-	for (def i : 2..numberOfButtons) {
+	for(def endpoint : 2..numberOfButtons) {
 		try {
-			String childDni = "${device.deviceNetworkId}:$i"
-			def componentLabel = (device.displayName.endsWith(' 1') ? device.displayName[0..-2] : device.displayName) + "${i}"
-			addChildDevice("Child Button", "${device.deviceNetworkId}:${i}", device.hubId,
-                	[completedSetup: true, label: "${device.displayName} button ${i}",
-                 	isComponent: true, componentName: "button$i", componentLabel: "Button $i"])
-		} catch (Exception e) {
+			String childDni = "${device.deviceNetworkId}:$endpoint"
+			def componentLabel = (device.displayName.endsWith(' 1') ? device.displayName[0..-2] : device.displayName) + "${endpoint}"
```

Comment:
I suggest calling this `childLabel` instead of `componentLabel` since it's not actually be used as the `componentLabel` on the component.

Reply:
updated, please have a review again